nexus/src/authz/omicron.polar

```diff
@@ -254,12 +254,25 @@ has_permission(actor: AuthenticatedActor, "modify", session: ConsoleSession)
 	if has_role(actor, "external-authenticator", session.fleet);
 
 resource SiloUser {
-	permissions = [ "read", "modify", "create_child" ];
-        relations = { parent_silo: Silo };
+	permissions = [
+	    "list_children",
+	    "modify",
+	    "read",
+	    "create_child",
+	];
+	roles = [ "admin", "collaborator", "viewer" ];
```

Comment:
I didn't mean to suggest adding roles here, but rather to write the rules below in terms of roles on the parent rather than permissions on the parent.

So I think you could drop the roles here, and the rules at L265-270, and then say stuff like:

```
"list_child" if "viewer" on "parent_silo"
"read" if "viewer" on "parent_silo"
"modify" if "admin" on "parent_silo"
"create_child" if "admin" on "parent_silo"
```

Reply:
Got it, thanks for the clarification. Fixed in 4036a060.